VK_V1.0/VK_V1.0/Servics/DataProvider.swift

```diff
@@ -0,0 +1,53 @@
+// DataProvider.swift
+// Copyright © RoadMap. All rights reserved.
+
+import RealmSwift
+
+/// Взаимодействие с базой данных
+final class DataProvider {
+    // MARK: - Public Methods
+
+    func saveFriendsInRealm(user: [User]) {
+        do {
+            let realm = try Realm()
+            try realm.write {
+                realm.add(user, update: .modified)
+            }
+        } catch {
+            print(error.localizedDescription)
+        }
+    }
+
+    func saveGroupsInRealm(group: [Group]) {
+        do {
+            let realm = try Realm()
+            try realm.write {
+                realm.add(group, update: .modified)
+            }
+        } catch {
+            print(error.localizedDescription)
+        }
+    }
+
+    func savePhotoData(photos: [Photo]) {
+        do {
+            let realm = try Realm()
+            try realm.write {
+                realm.add(photos)
+            }
+        } catch {
+            print(error.localizedDescription)
+        }
+    }
```

Comment:
```suggestion
    static func save<T: Object>(items: [T],
                                config: Realm.Configuration = Realm.Configuration(deleteRealmIfMigrationNeeded: true),
                                update: Bool = true) { 
        do {
            let realm = try Realm(configuration: self.deleteIfMigration)
            
            try realm.write {
                realm.add(items, update: true)
            }
            
        } catch {
            print(error. localizedDescription)
        }
    }
```
в важном файл  RealmService